app/graphql/types/partner_type.rb

```diff
@@ -2,32 +2,57 @@ module Types
   class PartnerType < Types::BaseObject
     
 
-    description 'A Partner who runs Events'
+    description 'Organisations that run events'
 
-    # placecalID
-    field :id, ID, null: false
-    field :name, String, null: false
-    field :summary, String
-    field :description, String
+    field :id, ID, 
+      null: false,
+      description: 'ID of partner'
 
-    field :accessibility_summary, String, method: :accessibility_info
+    field :name, String, 
+      null: false,
+      description: 'A short string about this partner, an alias for `summary`'
 
-    field :logo, String
-    field :address, AddressType
+    field :summary, String,
+      description: 'A short string describing partner'
 
-    field :url, String
-    field :facebook_page, String, method: :facebook_link
-    field :twitter_handle, String, method: :twitter_handle
+    field :description, String,
+      description: 'Longer text about partner with more detail'
 
-    field :areas_served, [NeighbourhoodType], method: :service_area_neighbourhoods
+    field :accessibility_summary, String,
+      method: :accessibility_info,
+      description: 'Accessibility information about this partner and the kind of events they run'
```

Comment:
Information about this Partner's accessibility

Reply:
again i feel you're describing the `accessibility_info` field as 'information about accessibility'

Reply:
erm is it not?

Reply:
I was trying to say something about the kind of information that this summary will contain. Accessibility Info (i think) is a bit vague? Is this describing how awesome the partner is about being disability positive or is this a description of the venues accessibility policy? guide dogs welcome, hearing impaired devices available, mobility scooter/wheelchair accessible?

So "Describes partner's accessibility capabilities" ? "A short text outlining accessibility options" ?

Reply:
hmmm i see your point. i mean on a literal level its to contain an accessibility statement which we hope will be based on the zine we wrong.

but like 'capabilities' is not quite the right word. this could say, for example, that they dont have wheelchair access - we care more that peope bothered to fill it in as people are often not in control of this.

it covers trans inclusion and neurodivergent inclusion too

maybe 'An accessibility statement written by this partner'? a bit literal but this could contain all kinds of things and not everyone organises events